docs/guidance/stable-diffusion-rayservice.md

```diff
@@ -0,0 +1,55 @@
+# Serve a StableDiffusion text-to-image model using RayService
+
+> **Note:** The Python files for the Ray Serve application and its client are in the [ray-project/serve_config_examples](https://github.com/ray-project/serve_config_examples) repo 
+and [the Ray documentation](https://docs.ray.io/en/latest/serve/tutorials/stable-diffusion.html).
+
+## Step 1: Create a Kubernetes cluster with GPUs
+
+You can follow [aws-eks-gpu-cluster.md](./aws-eks-gpu-cluster.md) to create an AWS EKS cluster with 1
+CPU (`m5.xlarge`) and 1 GPU (`g5.xlarge`) nodes.
+
+## Step 2: Install KubeRay operator
+
+Follow [this document](../../helm-chart/kuberay-operator/README.md) to install the nightly KubeRay operator via 
+Helm. Note that the YAML file in Step 3 uses `serveConfigV2`, which is first supported by KubeRay v0.6.0.
+
+## Step 3: Install a RayService
+
+```sh
+# path: ray-operator/config/samples/
+kubectl apply -f ray-service.stable-diffusion.yaml
+```
+
+* The `tolerations` for workers must match the taints on the GPU node group. Without the tolerations, worker Pods won't be scheduled on GPU nodes.
+    ```yaml
+    # Please add the following taints to the GPU node.
+    tolerations:
+        - key: "ray.io/node-type"
+        operator: "Equal"
+        value: "worker"
+        effect: "NoSchedule"
+    ```
+* Install `diffusers` in `runtime_env` as it is not included by default in the `ray-ml` image.
+
+## Step 4: Forward the port of Serve
+
+```sh
+kubectl port-forward svc/stable-diffusion-serve-svc 8000
+```
+
+Note that the Serve service will be created after the Serve applications are ready and running. This process may take approximately 1 minute after all Pods in the RayCluster are running.
```

Comment:
Is it the Kubernetes service that's created after the Serve apps are ready?
```suggestion
Note that the RayService's Kubernetes service will be created after the Serve applications are ready and running. This process may take approximately 1 minute after all Pods in the RayCluster are running.
```

Reply:
> Is it the Kubernetes service that's created after the Serve apps are ready? 

Yes.